src/js/components/Select/SelectContainer.js

```diff
@@ -42,6 +42,34 @@ const ClearButton = forwardRef(
     const align = position !== 'bottom' ? 'start' : 'center';
     const buttonLabel = label || `Clear ${name || 'selection'}`;
     const { passThemeFlag } = useThemeValue();
+    const buttonKind = theme.select.clear?.button;
+
+    if (buttonKind) {
+      // New structure when `theme.select.clear.button` is defined
+      const containerProps = theme.select.clear?.container || {};
+      const textProps = theme.select.clear?.text || {};
+
+      return (
+        <Box flex="grow" {...containerProps}>
+          <StyledButton
+            a11yTitle={`${buttonLabel}. Or, press ${
+              position === 'bottom' ? 'shift tab' : 'down arrow'
+            } to move to select options`}
+            fill="horizontal"
+            justify="start"
+            ref={ref}
+            onClick={onClear}
+            kind={theme.select.clear?.button}
+            label={<Text {...textProps}>{buttonLabel}</Text>}
+            align={align}
+            {...passThemeFlag}
+            {...rest}
+          />
+        </Box>
+      );
+    }
+
+    // Default structure when `kind` is not defined
     return (
       <StyledButton
         a11yTitle={`${buttonLabel}. Or, press ${
```

Comment:
Maybe we can pull out the repeated button props into an object that is spread across both buttons?

```
const buttonProps = {
   a11yTitle: `${buttonLabel} ...`,
   ref: ref,
   onClick: onClear,
   ....
}

Reply:
done